vms/platformvm/blocks/executor/proposal_block_test.go

```diff
@@ -100,7 +99,7 @@ func TestApricotProposalBlockTimeVerification(t *testing.T) {
 	}
 
 	// setup state to validate proposal block transaction
-	onParentAccept.EXPECT().CaminoGenesisState().Return(&genesis.Camino{}, nil)
```

Comment:
I don't like camino in here (as said in initial PR where this came in)
If this is not 100% required (and I doubt it is), pls, remove and use original license header

Reply:
its required, cause its called during test
and without this line, test will always fail

Reply:
I'll check this again to be sure that behaviour still the same
But I guess it is, cause if not, test would fail with "mock didnt call the expected func"

Reply:
Will be chnaged with next PR